plugins/modules/dedicated_server_networkinterfacecontroller.py

```diff
@@ -21,8 +21,11 @@
         description: The service_name
```

Comment:
Could you change the description?
```suggestion
        description: The service name
```